package.json

```diff
@@ -36,14 +36,13 @@
     "pretty-quick": "^3.1.0",
     "rimraf": "^3.0.2",
     "secret-stack": "^6.4.0",
-    "ssb-db2-box2": "^0.4.0",
     "ssb-caps": "^1.1.0",
-    "tap-bail": "^1.0.0",
-    "tap-spec": "^5.0.0",
+    "ssb-db2-box2": "^0.4.0",
+    "tap-arc": "^0.3.4",
     "tape": "^5.3.0"
   },
   "scripts": {
-    "test": "tape test/*.js | tap-bail | tap-spec",
```

Comment:
:hot_pepper: `tap-spec` generates npm audit high level warnings. I want these gone from repos because they add noise which makes it hard to see actual vulnerabilities.

I researched the problem with `tap-spec` and it's fixable, but this module is no longer maintained. I forked it, but it's not ideal. After a bunch of searching I've settled on `tap-arc` which has really nice diff output, "bail" as builtin functionality, and the developers are a company who are very responsive to PRs/ issues.